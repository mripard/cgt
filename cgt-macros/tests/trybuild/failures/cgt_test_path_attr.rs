#[cgt_macros::cgt_test_with_path(attribute)]
fn test(_: &Path) -> Result<(), cgt_core::TestError> {
    Ok(())
}

fn main() {}
