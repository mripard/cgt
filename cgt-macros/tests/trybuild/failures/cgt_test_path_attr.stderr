error: This macro doesn't use any attribute
 --> tests/trybuild/failures/cgt_test_path_attr.rs:1:34
  |
1 | #[cgt_macros::cgt_test_with_path(attribute)]
  |                                  ^^^^^^^^^
